Caboodle/Sms/Sms.uwp.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Windows.ApplicationModel.Chat;
+using Windows.Devices.Sms;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        static bool GetCanSendSms() => true;
+
+        static bool GetCanSendSmsInBackground() => true;
+
+        private static void PlatformSendSms(string recipient, string message)
+        {
+            var msg = new ChatMessage { Body = message };
+            if (!string.IsNullOrWhiteSpace(recipient))
+                msg.Recipients.Add(recipient);
+
+#pragma warning disable 4014
+            ChatMessageManager.ShowComposeSmsMessageAsync(msg);
```

Comment:
This is async... so really this should be a Task and the SendSms should be SendSmsAsync